### PR TITLE
[amends #89] Add badge to upper right when a component cannot be found

### PR DESCRIPTION
**Pull request by kevinmpowell**
@humancompanion, can you take a look?

Sorry for all the noise. Grunt wasn't working for me (was missing grunt-contrib-jshint, grunt-exec, load-grunt-tasks) when I tried to run it, so I npm install --save-dev'd those dependencies. I also added a new test: `loader-errors-test.html`

I couldn't add this to the other loader test suite since it's testing breakage which causes the other tests to fail.

### CODE REVIEW

**Comment by humancompanion** [comment]
@kevinmpowell - Sorry, I can't really find what changed in here. It looks like you unintentionally checked-in _site.

**Comment by kevinmpowell** [comment]
@humancompanion Let me try again, it'll be two pull requests...one to fix dependency issues, and then one to add the feature
